plugins/intrinsics/src/main/java/org/qbicc/plugin/intrinsics/core/CoreIntrinsics.java

```diff
@@ -263,6 +264,14 @@ public static void registerJavaLangSystemIntrinsics(CompilationContext ctxt) {
             ctxt.getLiteralFactory().literalOf(0);
 
         intrinsics.registerIntrinsic(systemDesc, "identityHashCode", objectToIntDesc, identityHashCode);
+
+        // registerNatives
+
+        Literal voidLiteral = ctxt.getLiteralFactory().zeroInitializerLiteralOfType(ctxt.getTypeSystem().getVoidType());
+
+        StaticIntrinsic registerNatives = (builder, target, arguments) -> voidLiteral;
+
+        intrinsics.registerIntrinsic(systemDesc, "registerNatives", emptyToVoid, registerNatives);
```

Comment:
It's trivial code so probably doesn't matter, but there's multiple copies of the empty `registerNatives` intrinsic.  If we need to add another, it might be better to commonize so there's only 1 copy of it

Reply:
I agree